src/MergeContext.js

```diff
@@ -108,6 +108,84 @@ class Approval {
     }
 }
 
+class StatusCheck
+{
+    constructor(raw) {
+        assert(raw.context);
+        assert(raw.state);
+        assert(raw.target_url);
+        assert(raw.description);
+
+        this.context = raw.context;
+        this.state = raw.state;
+        this.targetUrl = raw.target_url;
+        this.description = raw.description;
+
+        this._raw = raw;
```

Comment:
Remove as unused? I was thinking "context", "state", etc. would become getters for "_raw" so that you can still return and use the raw object elsewhere when needed, but if you do not need the raw object, then there is no need to keep a reference to it, I guess.

Reply:
Done.